src/Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN apk add --no-cache \
 ADD https://ftl.pi-hole.net/macvendor.db /macvendor.db
 COPY crontab.txt /crontab.txt
 
+# Add PADD to the container, too.
+ADD https://raw.githubusercontent.com/pi-hole/PADD/PADD_FTLv6/padd.sh /usr/local/bin/padd
```

Comment:
You may spare a layer using:
ADD --chmod=0755 https://raw.githubusercontent.com/pi-hole/PADD/PADD_FTLv6/padd.sh /usr/local/bin/padd

it could be applied an all ADD lines on the dockerfile. Changing owner is also possible:
ADD --chown=user:pihole --chmod=555 https://ftl.pi-hole.net/macvendor.db /macvendor.db

Reply:
Thanks. I added the alias and included the chmod.